apps/frontend/src/app/pages/profile/profile-edit/profile-edit.component.ts

```diff
@@ -118,46 +119,31 @@ export class ProfileEditComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.isAdmin = this.localUserService.isAdmin;
+    this.isModOrAdmin = this.localUserService.isModOrAdmin;
```

Comment:
Because `tap` was only firing when function in `switchMap` was returning local user subject. So you basically could edit as admin only if you are a local user. I don't know where else to put those